Dockerfile

```diff
@@ -217,6 +217,23 @@ RUN curl -sSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh -o
     && nvm alias default 14 \
     && npm install -g npm
 
+# Install PHP
+RUN apt-get install --assume-yes apt-transport-https software-properties-common \
+    && add-apt-repository ppa:ondrej/php \
+    && apt-get update \
+    && apt-get install --assume-yes \
+        php8.1 \
+        php8.1-cli \
+        php8.1-common \
+        php8.1-curl \
+        php8.1-mbstring\
+        php8.1-xml \
+    && php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" \
```

Comment:
Composer is standalone so we can replace with `COPY --from=composer:2 /usr/bin/composer /usr/bin/composer`

Reply:
What would do `COPY --from=composer:2 /usr/bin/composer /usr/bin/composer`? What is `--from=composer:2`?

Reply:
It seems that it downloads the whole docker image of composer
Isn't that heavier than downloading the composer-setup script?

Reply:
It is heavier yes; however it will be pulled optimistically so will be complete by the time that line is hit in the Dockerfile during build.

Either works, really :)